src/components/datagrid/theme.css

```
@@ -114,15 +114,21 @@
 .is-sortable {
   svg {
     margin-left: var(--spacer-smallest);
+    color: var(--color-teal-light);
```

Use a neutral color for the icon in an unsorted state (so it matches the text of the header cell), also use a lighter tint (than the text, which is in the 'darkest' tint) to indicate the sorting is inactive.

```suggestion
    color: var(--color-neutral-dark);
```